src/templates/static/legal/metatrader/terms_and_conditions.jsx

```diff
@@ -1,446 +1,663 @@
 import React from 'react';
 import BIEL from './biel.jsx';
 
-const Li = ({ title, subtitle, sub_list, paragraph }) => (
-    <li><strong>{title}</strong> &mdash; {subtitle}
-        { sub_list && <ol>{sub_list.map(list => <li key={list}>{list}</li>)}</ol> }
-        { paragraph && <p>{paragraph}</p> }
+const Li = ({ title, subtitle, sub_list, paragraph, children }) => (
+    <li><strong>{title}</strong> {subtitle && <span>&mdash; {subtitle}</span>}
+        {sub_list && <ol>{sub_list.map(list => <li key={list}>{list}</li>)}</ol>}
+        {paragraph && <p>{paragraph}</p>}
+        {children}
     </li>
 );
 
 const TermsAndConditions = () => (
     <div id='mt-tnc'>
-        <h2 data-anchor='introduction-and-scope-of-agreement'>{it.L('A. Introduction and Scope of Agreement')}</h2>
+        <h2 data-anchor='introduction-and-scope-of-agreement'>{it.L('A. Introduction and scope of agreement')}</h2>
         <ol>
             <li>
-                {it.L('Depending on the jurisdiction to which a Client\'s account has been attached in accordance with the account opening procedures of the website, the term \'we\', \'us\', \'our\', \'ours\', \'the Company\', or \'[_1]\' shall denote either', it.website_name)}&nbsp;
-                <span data-show='-eucountry'>{it.L('Binary (V) Ltd.,')}&nbsp;</span>
-                <span data-show='-eucountry'>{it.L('Binary (C.R.) S.A.,')}&nbsp;</span>
-                <span data-show='-eucountry'>{it.L('Binary (BVI) Ltd.,')}&nbsp;</span>
-                <span data-show='-eucountry'>{it.L('Binary (FX) Ltd.,')}&nbsp;</span>
-                <span>{it.L('Binary (Europe) Ltd, or')}&nbsp;</span>
-                <span>{it.L('Binary Investments (Europe) Ltd.')}</span>
+                {it.L('Depending on the jurisdiction to which a Client\'s account has been attached in accordance with the account opening procedures of the website, the terms \'the Company\' and \'[_1]\' shall denote either', it.website_name)}&nbsp;<br />
+                <ul>
+                    <span data-show='-eucountry'>{it.L('Binary (V) Ltd,')}&nbsp;<br /></span>
```

Comment:
is Ltd,; correct? o maybe Ltd. ?

Reply:
The document says Ltd,; which confused me as well, but I'm waiting for the copywriting guys to give their comments

Reply:
The semicolons are a no go; I removed them